.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 jobs:
     'test':
         docker: &DOCKERIMAGE
-            - image: circleci/node:8.11.3-stretch-browsers
+            - image: circleci/node:10.12.0-stretch-browsers
```

Comment:
I don't think it's a good idea to build ui on a newer node version than the projects that use it

Reply:
@sk91 I will change wallet to also use the latest node. It should work now because it works on my local dev env. Cheers

Reply:
cool, but pay attention to electron version as well.